### PR TITLE
Translate Ruby Snap post (pt)

### PR DESCRIPTION
**Pull request by vnbrs**

### CODE REVIEW

File: pt/news/_posts/2018-11-08-snap.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+```
+sudo snap switch ruby --channel=2.3/stable
+sudo snap refresh
+```
+
+Nosso snap seta as variáveis de ambiente `GEM_HOME` e `GEM_PATH`
+para `$HOME/.gem`.
+Então se você quer executar comandos instalados por gems, como `rails` e
+`rspec`, sem utilizar `bundle exec`, você precisará adicionar a seguinte
+linha no seu arquivo shell rc (como .bashrc`):
```
Comment (stomar) [inline pt/news/_posts/2018-11-08-snap.md]: missing "`"

**Comment by stomar** [comment]
@fuadsaud @jcserracampos @fpgentil @JuanitoFatas

File: pt/news/_posts/2018-11-08-snap.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+lang: pt
+---
+
+Nós oficialmente lançamos um pacote snap para a linguagem Ruby.
+
+<https://snapcraft.io/ruby>
+
+Snap é um sistema de pacotes desenvolvido pela Canonical. Ele permite que você
+distribua um software com suas dependências para diferentes sistemas Linux.
+Isso resolve o problema de quando um usuário não consegue isntalar a última
```
Comment (jcserracampos) [inline pt/news/_posts/2018-11-08-snap.md]:
```suggestion
Isso resolve o problema de quando um usuário não consegue instalar a última
```

File: pt/news/_posts/2018-11-08-snap.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+comando:
+
+```
+sudo snap install ruby --classic
+```
+
+(Se você utiliza outras distribuições Linux, por favor acesse
+<https://docs.snapcraft.io/installing-snapd/6735>.)
+
+Nosso snap utiliza a funcionalidade de "canal/channel" para distribuir
```
Comment (jcserracampos) [inline pt/news/_posts/2018-11-08-snap.md]:
```suggestion
Nosso snap utiliza a funcionalidade de "canal" para distribuir
```

File: pt/news/_posts/2018-11-08-snap.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+```
+
+Você pode também utilizar diversos canais. O seguinte comando muda para a versão 2.3:
+
+```
+sudo snap switch ruby --channel=2.3/stable
+sudo snap refresh
+```
+
+Nosso snap seta as variáveis de ambiente `GEM_HOME` e `GEM_PATH`
```
Comment (jcserracampos) [inline pt/news/_posts/2018-11-08-snap.md]:
```suggestion
Nosso snap configura as variáveis de ambiente `GEM_HOME` e `GEM_PATH`
```

Comment (jcserracampos) [reply pt/news/_posts/2018-11-08-snap.md]: Setar is not a portuguese verb.

**Comment by vnbrs** [comment]
:shipit:

**Comment by JuanitoFatas** [comment]
Thanks for your contribution @vnbrs  <3

**Pull request merged**